src/xmidtsend_rbus.c

```diff
@@ -436,7 +436,7 @@ void sendXmidtEventToServer(wrp_msg_t * msg, rbusMethodAsyncHandle_t asyncHandle
 	{
 		errorMsg = strdup("Memory allocation failed");
 		ParodusError("Memory allocation failed\n");
-		createOutParamsandSendAck(msg, asyncHandle, errorMsg, MSG_PROCESSING_FAILED);
+		createOutParamsandSendAck(msg, asyncHandle, errorMsg, MSG_PROCESSING_FAILED, RBUS_ERROR_INVALID_INPUT);
```

Comment:
same as above, here also issue happened due to provider processing failure .
so error code should be RBUS_ERROR_INVALID_RESPONSE_FROM_DESTINATION

Reply:
resolved in latest code